server/config/defaults.js

```diff
@@ -27,6 +27,7 @@ function getDefaults () {
     // core modules
     account: {},
     admin: {},
-    store: {}
+    store: {},
+    plugins: []
```

Comment:
so that will be an object now, right? With they keys being the plugin names, and values being either `true` (default values), `false` (disabled) or an object of options